pkg/cmd/attestation/verify/policy.go

```diff
@@ -25,26 +25,57 @@ func expandToGitHubURL(tenant, ownerOrRepo string) string {
 	return fmt.Sprintf("(?i)^https://%s.ghe.com/%s/", tenant, ownerOrRepo)
 }
 
-func buildSANMatcher(opts *Options) (verify.SubjectAlternativeNameMatcher, error) {
+func newEnforcementCriteria(opts *Options, a artifact.DigestedArtifact) (verification.EnforcementCriteria, error) {
+	c := verification.EnforcementCriteria{
```

Comment:
The Enforcement criteria maps Options to strings we're verifying. What does having artifact in here buy us?

Reply:
like, the contents of the `opts` will never modify the `DigestedArtifact`, right? so why is it EC's responsibility to be a container for it?